### PR TITLE
Update Distance.py

### PR DESCRIPTION
**Pull request by mrmucox**
Fixed Distance.get_inches(). Millimeters would return 1/10th the size, and Meters would always fail or return 'MISSING'.

### CODE REVIEW

File: tests/ish_report_test.py [hunk 1]
```diff
@@ -111,7 +111,7 @@ def test_fm15(self):
     self.assertEqual(weather.report_type, 'FM-15')
     self.assertEqual(weather.elevation, 205)
     self.assertEqual(weather.wind_speed, 1.5)
-    self.assertEqual(weather.wind_speed.get_miles(), 3.3554)
+    self.assertEqual(weather.wind_speed.get_MilesPerHour(), 3.3554)
```
Comment (ryaka) [inline tests/ish_report_test.py]: Can we change this to snake case to fit in with convention? get_miles_per_hour / get_mph

File: ish_parser/Speed.py [hunk 2]
```diff
@@ -1,18 +1,20 @@
...
+  between m/s and mph, yay '''
   METERSPERSECOND = 1
-  MILESPERHOUR = 1
+  MILESPERHOUR = 2
+  SPEED_SCALE = 10
+  MPH_PER_MPS = 100 / 2.54 / 12 / 5280 * 60 * 60
   MISSING = [999, 9999, 999.9]
 
-  def get_miles(self):
+  def get_MilesPerHour(self):
```
Comment (ryaka) [inline ish_parser/Speed.py]: Can we change this to snake case to fit in with convention? get_miles_per_hour / get_mph

File: .vscode/settings.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (ryaka) [inline .vscode/settings.json]: Remove this file

File: ish_parser/Components/PresentWeatherComponent.py [hunk 4]
```diff
@@ -2,26 +2,97 @@
...
-    self.present_weather_array = {'intensity': self.INTENSITY[string[0:1]],
-                                  'descriptor': self.DESCRIPTOR[string[1:2]],
-                                  'precipitation': self.PRECIP[string[2:4]]}
-    present = self.present_weather_array
+    self.present_weather_array = {
+      'intensity': self.INTENSITY[string[0:1]],
+      'descriptor': self.DESCRIPTOR[string[1:2]],
+      'precipitation': self.PRECIP[string[2:4]],
+      'obscuration': self.OBSCURATION[string[4:5]]
+      }
```
Comment (ryaka) [inline ish_parser/Components/PresentWeatherComponent.py]: Fix curly brace alignment

File: ish_parser/Components/PresentWeatherComponent.py [hunk 5]
```diff
@@ -2,26 +2,97 @@
...
+    self.present_weather_event_array =  {
+      'descriptor': self.EVENT_DESCRIPTOR[string[1:2]],
+      'precipitation': self.PRECIP[string[2:4]],
+      'obscuration': self.OBSCURATION[string[4:5]]
+      }
```
Comment (ryaka) [inline ish_parser/Components/PresentWeatherComponent.py]: Fix curly brace alignment

File: ish_parser/Components/PresentWeatherComponent.py [hunk 6]
```diff
@@ -2,26 +2,97 @@
...
+      
+    present = [
+      self.INTENSITY[string[0:1]],
+      self.DESCRIPTOR[string[1:2]],
+      self.PRECIP[string[2:4]],
+      self.OBSCURATION[string[4:5]]
+      ]
```
Comment (ryaka) [inline ish_parser/Components/PresentWeatherComponent.py]: Fix brace alignment

File: ish_parser/Components/PresentWeatherComponent.py [hunk 7]
```diff
@@ -2,26 +2,97 @@
...
+    present_event = [
+      self.EVENT_DESCRIPTOR[string[1:2]],
+      self.PRECIP[string[2:4]],
+      self.OBSCURATION[string[4:5]]
+      ]
```
Comment (ryaka) [inline ish_parser/Components/PresentWeatherComponent.py]: Fix brace alignment

File: ish_parser/Components/SkyCoverSummationComponent.py [hunk 8]
```diff
@@ -46,11 +55,19 @@ class SkyCoverSummationComponent(BaseComponent):
 
   def loads(self, string):
 
-    self.sky_cover_summation = {'coverage': Constant(string[0:1], None,
+    self.sky_cover_summation  = {'coverage': Constant(string[0:1], None,
```
Comment (ryaka) [inline ish_parser/Components/SkyCoverSummationComponent.py]: Remove extra space here

**Comment by haydenth** [comment]
Hey folks I am so frickin sorry - i literally just noticed this pull request and it got lost somehow on my end during covid and all that fun. Anyway, taking a look at this today and i'll merge in. I may edit.

**Comment by haydenth** [comment]
OK so this is failing tests, looks like because it's using a string vs constant.. Looking now

```
======================================================================
FAIL: test_string (tests.ComponentTest.SkyCoverSummationComponentTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/tom/projects/ish_parser/tests/ComponentTest.py", line 11, in test_string
    self.assertEqual(sd.sky_cover_summation, {
AssertionError: {'coverage': OVERCAST - 8/8 coverage, 'coverage_simple[80 chars]sing} != {'coverage': 'OVERCAST - 8/8 coverage', 'secondary_cov[57 chars]ing'}
- {'characteristic': Missing,
+ {'characteristic': 'Missing',
?                    +       +

-  'coverage': OVERCAST - 8/8 coverage,
+  'coverage': 'OVERCAST - 8/8 coverage',
?              +                       +

-  'coverage_simple': Overcast,
   'height': 610,
-  'secondary_coverage': Missing}
+  'secondary_coverage': 'Missing'}

```

**Comment by haydenth** [comment]
tests pass now

```
~/projects/ish_parser $ python3 -m unittest tests
.........................
Weather Station: 725300 (41.995, -87.934)
Elevation: 205 m
Time: 2015-01-09 00:39:00+00:00 UTC
Air Temperature: -10.6 C (12.9 F)
Wind Speed: 7.2 m/s (16.1059 mph)
Wind Direction: 220
Present Weather Obs: ['Light Snow', 'Moderate Blowing Snow']
Precipitation: [{'hours': 1, 'depth': 0.0}]
Cloud Coverage: [{'coverage': 08, 'base_height': 610, 'cloud_type': '59'}] oktas
Cloud Summation: [{'coverage': 'OVERCAST - 8/8 coverage', 'coverage_simple': 'Overcast', 'secondary_coverage': 'Missing', 'height': 610, 'characteristic': 'Missing'}]
Solar Irradiance: None 
    
.........
Weather Station: 724666 (39.567, -104.85)
Elevation: 1793 m
Time: 2004-04-27 20:00:00+00:00 UTC
Air Temperature: MISSING C (MISSING F)
Wind Speed: MISSING m/s (None mph)
Wind Direction: MISSING
Present Weather Obs: None
Precipitation: None
Cloud Coverage: None oktas
Cloud Summation: None
Solar Irradiance: [{'time_period': 0060, 'global_irradiance': 1014, 'irradiance_data_flag': 'Value estimated; passes all pertinent SERI_QC tests', 'direct_beam_irradiance': 0989, 'direct_beam_irradiance_data_flag': 'Passed two-component test; data fall within 0.03 of the Gompertz boundaries', 'diffuse_irradiance': 0131, 'diffuse_irradiance_data_flag': 'Passed two-component test; data fall within 0.03 of the Gompertz boundaries', 'uvb_global_irradiance': 9999}] 
    
........
----------------------------------------------------------------------
Ran 42 tests in 2.672s

OK

```

Thanks for fixing this @mrmucox

**Pull request merged**